server/unittools.cpp

```diff
@@ -2586,53 +2578,41 @@ void kill_unit(struct unit *pkiller, struct unit *punit, bool vet)
           notify_player(player_by_number(i), ptile, E_UNIT_LOST_DEF,
                         ftc_server,
                         // TRANS: "Cannon ... the Polish Destroyer."
-                        _("%s lost to an attack by the %s %s."), punit_link,
-                        nation_adjective_for_player(pvictor), pkiller_link);
+                        _("%s lost when the %s %s defeated your %s."),
+                        punit_link, nation_adjective_for_player(pvictor),
+                        pkiller_link, punit_link);
         } else {
           fc_assert(other_killed[i] != punit);
           notify_player(player_by_number(i), ptile, E_UNIT_LOST_DEF,
                         ftc_server,
                         /* TRANS: "Cannon lost when the Polish Destroyer
                          * attacked the German Musketeers." */
-                        _("%s lost when the %s %s attacked the %s %s."),
+                        _("%s lost when the %s %s defeated the %s %s."),
                         unit_link(other_killed[i]),
                         nation_adjective_for_player(pvictor), pkiller_link,
                         nation_adjective_for_player(pvictim), punit_link);
         }
       } else if (num_killed[i] > 1) {
         if (i == player_index(pvictim)) {
           int others = num_killed[i] - 1;
-
-          if (others == 1) {
-            notify_player(
-                player_by_number(i), ptile, E_UNIT_LOST_DEF, ftc_server,
-                /* TRANS: "Musketeers (and Cannon) lost to an
-                 * attack from the Polish Destroyer." */
-                _("%s (and %s) lost to an attack from the %s %s."),
-                punit_link, unit_link(other_killed[i]),
-                nation_adjective_for_player(pvictor), pkiller_link);
-          } else {
-            notify_player(
-                player_by_number(i), ptile, E_UNIT_LOST_DEF, ftc_server,
-                /* TRANS: "Musketeers and 3 other units lost to
-                 * an attack from the Polish Destroyer."
-                 * (only happens with at least 2 other units) */
-                PL_("%s and %d other unit lost to an attack "
-                    "from the %s %s.",
-                    "%s and %d other units lost to an attack "
-                    "from the %s %s.",
-                    others),
-                punit_link, others, nation_adjective_for_player(pvictor),
-                pkiller_link);
-          }
+          notify_player(
+              player_by_number(i), ptile, E_UNIT_LOST_DEF, ftc_server,
+              /* TRANS: "Musketeers and 3 other units lost to
+               * an attack from the Polish Destroyer."
+               * (only happens with at least 2 other units) */
+              PL_("%d other unit lost when the %s %s defeated your %s.",
+                  "%d other units lost when the %s %s defeated your %s.",
```

Comment:
"3 other units" or "3 more units"?

Reply:
I think both work here? Other sounds better than more, to me. but I'm not native :)